test/e2e/tests/compression.go

```diff
@@ -19,6 +19,7 @@ import (
 	"testing"
 
 	"github.com/andybalholm/brotli"
+	"github.com/klauspost/compress/zstd"
```

Comment:
Is this a new package? I think we need to `go mod tidy`. Then also run make lint to check for linting errors.